src/worksheet.c

```diff
@@ -1507,7 +1507,8 @@ _worksheet_write_optimized_sheet_data(lxw_worksheet *self)
         while (read_size) {
             read_size =
                 fread(buffer, 1, LXW_BUFFER_SIZE, self->optimize_tmpfile);
-            fwrite(buffer, 1, read_size, self->file);
+            /* Ignore return value, no way to report error here */
```

Comment:
If you are respinning this patch can you add a full stop to the end of this sentence. That is my convention.